packages/backend/src/dbt/dbtCliClient.ts

```diff
@@ -46,8 +46,13 @@ export const getDbtConfig = async (
     if (!isRawDbtConfig(config)) {
         throw new Error('dbt_project.yml not valid');
     }
+    const updatedConfig = {
+        ...config,
+        'target-path': 'target',
```

Comment:
I'm not sure I understand the fix. Are you replacing the dynamic variable with a hardcoded `target` value ?

Reply:
Yes!

So this property controls where the output is written when dbt runs. since we run this on the server then it's better we control exactly where the output ends up. 

The only time this makes a difference will be when users mount in a dbt project with a local docker setup. In that case this will actually alter their dbt_project.yml - this is pretty uncommon though 

We use that second approach for our dev environments